new/detector/composition/javascript/testdata/testcases/object-deconstructing/deconstructing.js

```diff
@@ -0,0 +1,2 @@
+const { user } = req.params;
```

Comment:
Do we support things like `const { firstName,  lastName } = req.params` as well?
It's not clear to be based on what I can see

Reply:
we do, I've added test case for it

Reply:
I don't see the test case :/
Could you send me the link to it please?